src/components/UserStatistics/index.jsx

```diff
@@ -105,28 +106,42 @@ class AccountStatistics extends Component {
     if (this.state.updatedUserStats.length !== 0) {
       rows = this.state.updatedUserStats
     }
+
+    const compareRowsByDateDesc =(a, b) => {
+      return Moment(b.day).diff(Moment(a.day))
+    }
+
     ReactChartkick.addAdapter(Chart)
-    let chartData = {}
-    rows.map((item) =>
-      chartData[item.day] = item.newFollowers
+    let followersFollowingChartData = [{name: 'Followers', data: {}}, {name: 'Following', data: {}}]
+    let newFollowersChartData = [{name: 'New Followers', data: {}}]
+    rows.forEach((item) => {
+        followersFollowingChartData[0]['data'][item.day] = item.followers
+        followersFollowingChartData[1]['data'][item.day] = item.following
+        newFollowersChartData[0]['data'][item.day] = item.newFollowers
+      }
     )
+
     return (
       <div className={classes.wrapper}>
         <h1>User Statistics</h1>
+        <Paper>
+          <LineChart data={followersFollowingChartData} />
+          <LineChart data={newFollowersChartData} />
+        </Paper>
```

Comment:
Moved the charts to the top of the page and added the new following/followers chart